test/server/index.js

```diff
@@ -39,6 +58,19 @@ const WEB_PORT = process.env.WEB_PORT || 3031;
 const websiteDomain = `http://localhost:${WEB_PORT}`;
 let latestURLWithToken = "";
 
+let deviceStore = new Map();
+function saveCode({ email, phoneNumber, preAuthSessionId, urlWithLinkCode, userInputCode }) {
+    const device = deviceStore.get(preAuthSessionId) || {
+        preAuthSessionId,
+        codes: [],
+    };
+    device.codes.push({
+        urlWithLinkCode,
+        userInputCode,
+    });
+    console.log({ email, phoneNumber, urlWithLinkCode, userInputCode });
```

Comment:
You should remove this console.log and others from this file if they are not needed.